src/sdk/env.rs

```diff
@@ -0,0 +1,88 @@
+//! EnvResourceDetector
+//!
+//! Implementation of ResourceDetector in extract information from environment.
+use crate::sdk::resource::ResourceDetector;
+use crate::sdk::Resource;
+use crate::api::{KeyValue, Key, Value};
+use std::env;
+use std::time::Duration;
+
+const OTEL_RESOURCE_ATTRIBUTES: &str = "OTEL_RESOURCE_ATTRIBUTES";
+
+/// Resource detector implements ResourceDetector and is used to extractor general SDK configuration from environment
+/// See https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#telemetry-sdk for semantic conventions.
```

Comment:
Can we wrap these to 80 characters? rustfmt currently doesn't do this automatically, but the style guide for comments in rust seems to be 80 currently https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md#comments

Reply:
Sure thing. Don't think I can break the url but will wrap other comments.